airflow/www/static/css/_page_dags.scss

```diff
@@ -0,0 +1,46 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+.edit-dag {
+  width: 10px;
+}
+.actions {
+  display:flex;
+  flex-direction:row;
+  justify-content:space-around;
+}
+.recent-tasks {
+  padding: 0px !important;
+  width: 200px;
+  height: 10px;
+}
+.recent-tasks svg {
+  display: block;
+}
+.dags-runs {
+  padding: 0px !important;
```

Comment:
Do we have to have important here?

Reply:
Yes. I'ts required.
![Screenshot 2019-03-21 at 11 43 45](https://user-images.githubusercontent.com/12058428/54747310-d0352200-4bce-11e9-8719-9bc6eb1dcfce.png)